src/ru/skypro/task1/Book.java

```diff
@@ -0,0 +1,30 @@
+package ru.skypro.task1;
+
+public class Book {
+    String name;
+    int yearOfPublication;
+    String Author;
+
+
+    public Book(String name, String Author, int yearOfPublication) {
```

Comment:
```suggestion
    public Book(String name, Author author, int yearOfPublication) {
```
- переменные и поля именуются с маленькой буквы.
- в конструктор нужно передавать автора целиком